src/main/java/org/jfrog/hudson/pipeline/scripted/steps/DockerPullStep.java

```diff
@@ -74,15 +79,18 @@ protected BuildInfo run() throws Exception {
                 getContext().onFailure(new MissingArgumentException("Missing 'image' parameter"));
```

Comment:
Please avoid MissingArgumentException. See: https://github.com/jfrog/jenkins-artifactory-plugin/pull/327
```suggestion
                getContext().onFailure(new AbortException("Missing 'image' parameter"));
```